dev/helpers/mssql/read_required_migrations.sql

```diff
@@ -0,0 +1,20 @@
+CREATE OR ALTER PROCEDURE ReadRequiredMigrations
+    @migrationsFile NVARCHAR(MAX)
+AS
+BEGIN
+    SET NOCOUNT ON;
+
+    CREATE TABLE #InputMigrations (Filename NVARCHAR(MAX))
+
+    DECLARE @bulkInsertSql NVARCHAR(4000) = 'BULK INSERT #InputMigrations FROM ''' + @migrationsFile + ''' WITH (FIELDTERMINATOR = '';'', ROWTERMINATOR = '';'')';
+    EXEC(@bulkInsertSql);
```

Comment:
ℹ️ SQL injection attack 😜 ... not really a thing.

Reply:
Yes I figured it's not a risk here, but I should probably add a comment to this effect otherwise we'll be paying out a bounty to someone who thinks they've hacked our systems